commands/secret_create.go

```diff
@@ -40,6 +41,13 @@ func init() {
 }
 
 func preRunSecretCreate(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return fmt.Errorf("give a name of a secret")
+	}
+
+	if len(args) > 1 {
+		return fmt.Errorf("give ONLY the name of a single secret")
```

Comment:
this might read better as "too many values for secret name"

Reply:
That was me I think